qt_ui/windows/basemenu/NewUnitTransferDialog.py

```diff
@@ -36,13 +36,8 @@ def __init__(self, game: Game, origin: ControlPoint) -> None:
         self.game = game
         self.origin = origin
 
-        for cp in self.game.theater.controlpoints:
-            if (
-                cp != self.origin
-                and cp.is_friendly(to_player=True)
-                and cp.can_deploy_ground_units
-            ):
-                self.addItem(cp.name, cp)
+        for cp in self.game.blue.transit_network.connections_from(origin):
```

Comment:
`connections_from` only returns immediate neighbors. You want `has_path_between`.